recipes/r-mimic-demo/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mimic-demo
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/eth-mds/mimic-demo/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0491c29e3bffacb36e7a87f7896652cd0bc033c6411993a765a6223e5c4fc516
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rlang
+  run:
+    - r-base
+    - r-rlang
+
+test:
+  commands:
+    - $R -e "library('mimic.demo')"           # [not win]
+    - "\"%R%\" -e \"library('mimic.demo')\""  # [win]
+
+about:
+  home: https://github.com/eth-mds/mimic-demo
+  license: ODbL v1.0
+  license_file: LICENSE.md
+  summary: |
+    As an Enhances dependency to the CRAN R package ricu, this data package
+    provides the MIMIC-III demo data set. Due to CRAN size restrictions, this cannot be
+    served from CRAN but rather can be installed from a GitHub-hosted drat repository.
```

Use full description

```suggestion
    MIMIC-III (Medical Information Mart for Intensive Care III) is a
    large, freely-available database comprising deidentified health-related
    data associated with over 40,000 patients who stayed in critical care
    units of the Beth Israel Deaconess Medical Center between 2001 and 2012.
    The database includes information such as demographics, vital sign
    measurements made at the bedside (~1 data point per hour), laboratory test
    results, procedures, medications, caregiver notes, imaging reports, and
    mortality (both in and out of hospital). An open-access subset of
    MIMIC-III, comprising 100 randomly selected patients from the subset of
    patients who eventually die, is distributed under the Open Database
    License (ODbL) v1.0 and is provided as R package for convenience.
```